src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -85,14 +85,16 @@ const calcDonorCoreEntityStats = (
 
     if (tumorAndNormalExists) {
       coreStats[schemaNameToCoreCompletenessStat[clinicalType]] =
-        donor.specimens.map(sp => sp.clinicalInfo).filter(notEmpty).length / donor.specimens.length;
+        donor.specimens
+          .filter(specimen => specimen && specimen.sampleType?.includes('DNA'))
```

Comment:
I would change this to specify which sample types are included in the filter like so 

```
const dnaSampleTypes = ['ctDNA', ...];
...
specimen && specimen.sampleType && dnaSampleTypes.includes(specimen.sampleType);
```

Reply:
Sounds good, updated + tested. I'm guessing adding new DNA sample types is not something that happens often, i.e. this definition won't need to be dynamic.

Reply:
we'll have to update the code if sampleTypes get updated. it's good to not rely too much on naming conventions :)